File: C/Assignment 3/prime.c
// prime or not
//#include<stdio.h>
//void main()
//{
//	int num=7;
//	int i=2;
//	int prime=1;
//	while(i<num)
//	{
//		if(num%i==0)
//	        prime=0;
//	i++;
//	}
//	if(prime==1)
//        printf("the number is prime");
//    else
//        printf("the number is not prime");
//}

//OR
#include<stdio.h>
void main()
{
	int no=19;
	int i=2;
	while(i<no)
	{
		if(no%i==0)
		{
		    printf("not prime");
		    break;
	    }
	i++;
	}
	if(i==no)
	    printf("prime");
}

File: OOP/Assignment 4/Prime.java
class Prime
{
public static void main(String args[])
{
	int no=10;
	if(no==1 || no==0)
	    System.out.printf("%d is not prime",no);
	else
	{
		for(int start=2;start<=no;start++)
		{
			int prime=0;
			for(int i=2;i<start;i++)
			{
				if(start%i==0)
				    prime=1;
			}
			if(prime==0)
			     System.out.printf("%d ",start);
		}
	}
}
}